documentation/source/overview/references.rst

```diff
@@ -117,95 +111,193 @@ Applications
--  `Azzarito et al. Tracking the neurodegenerative gradient after spinal cord injury. NeuroImage Clinical, 2020 <https://pubmed.ncbi.nlm.nih.gov/32145681/>`_
--  `Querin et al. Development of new outcome measures for adult SMA type III and IV: a multimodal longitudinal study. J Neurol 2021 <https://pubmed.ncbi.nlm.nih.gov/33388927/>`_
--  `McLachlin et al. Spatial correspondence of spinal cord white matter tracts using diffusion tensor imaging, fibre tractography, and atlas-based segmentation. Neuroradiology 2021 <https://link.springer.com/article/10.1007/s00234-021-02635-9>`_
--  `Dvorak et al. Comparison of multi echo T2 relaxation and steady state approaches for myelin imaging in the central nervous system. Scientific reports 2021 <https://www.nature.com/articles/s41598-020-80585-7>`_
--  `Adanyeguh et al. Multiparametric in vivo analyses of the brain and spine identify structural and metabolic biomarkers in men with adrenomyeloneuropathy. NeuroImage: Clinical, 2021 <https://www.sciencedirect.com/science/article/pii/S2213158221000103>`_
--  `Meyer et al. Optimized cervical spinal cord perfusion MRI after traumatic injury in the rat. J. of Cerebral Blood Flow & Metabolism, 2021 <https://journals.sagepub.com/doi/10.1177/0271678X20982396>`_
--  `Solanes et al. 3D patient-specific spinal cord computational model for SCS management: potential clinical applications. Journal of Neural Engineering, 2021 <https://pubmed.ncbi.nlm.nih.gov/33556926/>`_
--  `Johnson et al. Changes in White Matter of the Cervical Spinal Cord after a Single Season of Collegiate Football. Neurotrauma Reports, 2021 <https://www.liebertpub.com/doi/10.1089/neur.2020.0035>`_
--  `Ost et al. Spinal Cord Morphology in Degenerative Cervical Myelopathy Patients; Assessing Key Morphological Characteristics Using Machine Vision Tools. Journal of Clinical Medicine, 2021 <https://www.mdpi.com/2077-0383/10/4/892>`_
--  `Lee et al. Diffusion‐prepared fast spin echo for artifact‐free spinal cord imaging. Magnetic Resonance in Medicine, 2021 <https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28751>`_
--  `Tinnermann et al. Observation of others’ painful heat stimulation involves responses in the spinal cord. Science Advances, 2021 <https://pubmed.ncbi.nlm.nih.gov/33789899/>`_
--  `Zhang et al. Automatic spinal cord segmentation from axial-view MRI slices using CNN with grayscale regularized active contour propagation. Computers in Biology and Medicine, 2021 <https://pubmed.ncbi.nlm.nih.gov/33780869/>`_
--  `Savini et al. Pilot Study on Quantitative Cervical Cord and Muscular MRI in Spinal Muscular Atrophy: Promising Biomarkers of Disease Evolution and Treatment? Front Neurol, 2021 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8039452/>`_
\ No newline at end of file
+.. list-table::
+   :widths: 80 20
+   :header-rows: 1
+
+   * - Citation
+     - Year
```

Comment:
Some small comments on the table:

* #3349 separates this section off into its own page (`studies.rst`). So, if changes are made in `references.rst`, they'll conflict.
* The year column is a bit wider than it maybe needs to be.
![Screenshot from 2021-04-24 16-21-56](https://user-images.githubusercontent.com/16181459/115971914-54c69d00-a519-11eb-8fc8-f570645891b0.png)
* Right now, the year info is duplicated, I think.
* What do you think of having the columns be Author, Title, Journal/Conference, and Year?

(Apologies if this is a WIP and my comments are premature here. :sweat_smile:)

Reply:
Update: The table idea was nixed (due to wanting to adhere to academic-style reference formatting), but some alternate formatting ideas were proposed in https://github.com/neuropoly/spinalcordtoolbox/issues/3357

Reply:
Ah okay, I removed the table, and perhaps we can do reformatting in a separate pull request?